src/assembly.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should avoid introducing a new directory 'src' in the current structure, please move it probably into the parent directory.

Reply:
I didn't put there but it's not related to the Java SDK. It's more generic and inherited from the Apache POM.

Reply:
well then put it in the parent directory but wait before re pushing I have some some secs I have some extra things I found.

Reply:
What `parent` directory ? I saw that a `release` folder has been created, I can eventually put there.

Reply:
OK, I move the assembly in the root folder.